resources/documentation/stories.rst

```diff
@@ -0,0 +1,467 @@
+
+To do so, use the `--check --diff` arguments with the `ansible-playbook`
+command.
+
+For example:
+
+.. code-block:: text
+
+  ansible-playbook /etc/bluebanquise/playbooks/managements.yml --diff --check
+
+2. If you have multiple managements nodes, only update them one by one, using
+the `--limit` argument, specifying the nodes each time. This can be combined
+with the dry run seen in 1.
+
+For example:
+
+.. code-block:: text
+
+  ansible-playbook /etc/bluebanquise/playbooks/managements.yml --limit management1
+  ansible-playbook /etc/bluebanquise/playbooks/managements.yml --limit management2
+  ansible-playbook /etc/bluebanquise/playbooks/managements.yml --limit management2
+
+Nodes
+=====
+
+Adding a new node
+-----------------
+
+To add a new node, go into the /etc/bluebanquise/inventory/cluster/nodes folder.
+
+Then here, find the file related to the master group of the new node. If you
+need to create a new master group, refer to the related story bellow.
+
+Open the file, and find the equipment_profile related to this node. If you
+need to create a new equipment_profile group, refer to the related story bellow.
+
+Now, simply add the node under the equipment_profile group, under *hosts*:
+
+.. code-block:: yaml
+
+  mg_computes:
+    children:
+      equipment_typeC:
+        hosts:
+          c001:  <<<< my new node
+
+You may also wish to add some network to the node. To do so, add a
+network_interfaces list this way.
+
+.. code-block:: yaml
+
+  mg_computes:
+    children:
+      equipment_typeC:
+        hosts:
+          c001:
+            bmc:                        # This instruction defines an attached BMC
+              name: bmanagement1        # This is the hostname of the BMC
+              ip4: 10.10.3.1            # This is the ipv4 of the BMC
+              mac: 08:00:27:dc:f8:f6    # This is the MAC hardware address of the BMC (for DHCP)
+              network: ice1-1           # This is the logical network this interface is connected to. Logical networks will be seen later.
+            network_interfaces:         # This is an instruction, to define bellow all host's NIC (Network Interface Controllers)
+              - interface: enp0s3       # This is the NIC name ('ip a' command to get NIC list)
+                ip4: 10.10.3.1          # This is the expected ipv4 for this NIC
+                mac: 08:00:27:dc:f8:f5  # This is the NIC MAC address, for the DHCP
+                network: ice1-1         # This is the logical network this NIC is linked to
+              - interface: ib0          # This is another NIC, not in the dhcp so no MAC is provided
+                ip4: 10.20.3.1
+                network: interconnect-1
+
+Then use the ansible-inventory command to check that the new host is listed on
+the configuration and seen by Ansible:
+
+.. code-block:: text
+
+  [root@management1 ~]# ansible-inventory --graph
+  [WARNING]: Unable to parse /etc/bluebanquise/internal as an inventory source
```

Comment:
Can you remove this warning from documentation? (if possible, remove it completely :stuck_out_tongue: )

Reply:
Done. I wonder if this would need to open a case to the Ansible team.

Reply:
No, this is an expected warning since there is no host in the inventory.

Workaround:
```
# cat > /etc/bluebanquise/internal/dummy << EOF
[internal]
dummy
EOF

# ansible-inventory --graph
@all:
  |--@internal:
  |  |--dummy
  |--@mg_computes:
  |  |--@equipment_typeC:
  |  |  |--c001
  |  |  |--c002
  |  |  |--c003
  |  |  |--c004
  |--@mg_logins:
  |  |--@equipment_typeL:
  |  |  |--login1
  |--@mg_managements:
  |  |--@equipment_typeM:
  |  |  |--management1
  |  |  |--management2
  |--@rack_1:
  |  |--c001
  |  |--c002
  |  |--c003
  |  |--c004
  |  |--login1
  |--@ungrouped:
```